python/paddle/device/__init__.py

```diff
@@ -568,16 +570,20 @@ def __repr__(self):
 
 class Stream:
     '''
+
     A device stream wrapper around StreamBase.
-    Parameters:
+
+    Args:
         device(str|paddle.CUDAPlace(n)|paddle.CustomPlace(n)): Which device the stream runn on. If device is None, the device is the current device. Default: None.
             It can be ``gpu``, ``gpu:x``,``custom_device``, ``custom_device:x``, where ``custom_device`` is the name of CustomDevicec,
```

Comment:
```suggestion
            It can be ``gpu``, ``gpu:x``, ``custom_device``, ``custom_device:x``, where ``custom_device`` is the name of CustomDevicec,
```